### ISSUE TITLE
Локализация

### ISSUE THREAD

**Issue by emailfortesting000001** [lamantin-group/react-native-yandex-payment#9]
Для ios всегда на английском. Так и должно быть? Или есть способ сменить на русский?

**Comment by ilya-raevskiy** [comment]
Появилась возможность сделать русский язык?

**Comment by novseje** [comment]
> Появилась возможность сделать русский язык?

Нет. В файлах модуля (не помню Яндекс Кассы или react-native-yandex-payment) есть языковые файлы. Мы просто английский заменили русским. При обновлении модулей этот файл перетрётся. Но в нашем случае это было приемлемым решением. Модули обновляются не часто, можно заменить после обновления.

**Comment by whalemare** [comment]
Мы делали точно также, к слову. Путей добавить локализацию я не нашел, даже issue оставлял у яндекса
